apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/modal.scss

```diff
@@ -0,0 +1,83 @@
+@import "@automattic/typography/styles/fonts";
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+/* stylelint-disable declaration-property-unit-allowed-list */
```

Comment:
What is breaking this? Is it the [`font-size`](https://github.com/Automattic/wp-calypso/blob/d9ab34a6013e40f7f9b474cba24a555bdc037566/.stylelintrc#L5)?
I've always been bad at `rem`, but I wonder if we should try to approximate the fixed `px` to the closest `rem` possible. 🤔

Reply:
Yup, it was added. I changed the font sizes to `rem` in 1d38851